spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -419,6 +419,12 @@ public class RolesEndpoint {
 
 ```
 
+If there is no JWT in the request, an empty `JsonWebToken` is injected.  Note that MP JWT will still perform
+Authentication and Authorization if the endpoint requires these verifications. Effectively, the injected empty token
+is only visible on unauthenticated and unauthorized endpoints.
+
+All method calls to the empty token return `null`.
```

Comment:
I suggest to move this next to the empty 'JsonWebToken'. 
an empty `JsonWebToken` is injected. =>
an empty `JsonWebToken` is injected, which means all method calls to this token return `null`.